integration-tests/specs/log.test.ts

```diff
@@ -0,0 +1,22 @@
+import {getLogPayloads} from "../helpers/embrace_mock_api";
+import {endSession} from "../helpers/session";
+import {EMBRACE_INTERNAL_LOG_SEVERITY_NUMBER} from "../helpers/log";
+import {ACTION_BUTTONS} from "../helpers/action";
+
+describe("Logs", () => {
+  it("should 3 logs (error | info | warn) be recorded as foreground", async () => {
+    await ACTION_BUTTONS.LOG.INFO();
+    await ACTION_BUTTONS.LOG.ERROR();
+    await ACTION_BUTTONS.LOG.WARN();
+    await new Promise(r => setTimeout(r, 1000));
+    await endSession();
+
+    const logPayloads = await getLogPayloads();
+    expect(logPayloads).toHaveLength(1);
+    const [logPayload] = logPayloads;
+
+    Object.values(EMBRACE_INTERNAL_LOG_SEVERITY_NUMBER).forEach(logSeverity =>
```

Comment:
I'd prefer to structure these like the existing specs we have for spans where as much as possible we assert against written out literal json, e.g.:

```
expect(logs).toEqual([
  {
    body: "my error log",
    severity: "info",
    time_unix_nano: logs[0].time_unix_nano,
    ...
  },
  etc.
]);
```

can add sorting during parsing to make sure we always have the same structure and for attributes that we don't care about the values or are dynamic can do something like `time_unix_nano: logs[0].time_unix_nano` to reflect the value back or add helpers like we have with `commonEmbraceSpanAttributes`